metricbeat/module/ceph/pool_disk/pool_disk_integration_test.go

```diff
@@ -22,13 +22,20 @@ import (
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
 func TestData(t *testing.T) {
```

Comment:
We have realized that this call `mbtest.ReportingFetchV2(f)` must be omitted here because this test must not be launched in CI and check is happening in `mbtest.WriteEventsReporterV2` so it should end up looking more or less like this:

```go
func TestData(t *testing.T) {
	f := mbtest.NewReportingMetricSetV2(t, getConfig())

	if err := mbtest.WriteEventsReporterV2(f, t, ""); err != nil {
		t.Fatal("write", err)
	}
}
```

This is the only important change here to avoid a flaky test that was happening